interface/eaw_ideas.gfx

```diff
@@ -3361,11 +3365,16 @@ spriteTypes = {
 		name = "GFX_idea_FEZ_medium_tank_manufacturer_advisor"
 		texturefile = "gfx/interface/ideas/FEZ/FEZ_medium_tank_manufacturer_advisor.tga"
 	}
-	#BUF - Concerns#
+	#NCH - Concerns#
 	SpriteType = {
 		name = "GFX_idea_BUF_tank_factory"
 		texturefile = "gfx/interface/ideas/BUF/BUF_tank_factory.tga"
 	}
+	#BUF - Concerns#
+	SpriteType = {
+		name = "GFX_idea_NCH_generic_tank_manufacturer"
+		texturefile = "gfx/interface/ideas/NCH/NCH_generic_tank_manufacturer.tga"
+	}
```

Comment:
Seems like you copy-pasted here then edited the wrong one. You have NCH under BUF comment and a BUF one under NCH.